GramAddict/core/log.py

```diff
@@ -54,13 +54,15 @@ def create_log_file_handler(filename):
     return file_handler
 
 
-def configure_logger():
+def configure_logger(args):
     global g_session_id
     global g_log_file_name
     global g_logs_dir
     global g_file_handler
     global g_log_file_updated
 
+    console_level = logging.DEBUG if "--debug" in args else logging.INFO
```

Comment:
The new parameter you added has an action to `store_true` if the argument `--debug`is found. No need to hardcode the string `--debug`here. I think checking `args.debug`is enough.

`console_level = logging.DEBUG if args.debug else logging.INFO`

Please, double check what I said, but I'm quite sure.

Reply:
The reason why is because we configure logging first thing before we even process args. On purpose. I don't think we should parse args before we configure logging.

Reply:
Ahaaaa. I get it.